readthedocs/vcs_support/backends/git.py

```diff
@@ -60,21 +60,37 @@ def fetch(self):
         if code != 0:
             raise RepositoryError
 
-    def checkout_revision(self, revision=None):
+    def checkout_revision(self, revision=None, env=None):
         if not revision:
             branch = self.default_branch or self.fallback_branch
             revision = 'origin/%s' % branch
 
-        code, out, err = self.run('git', 'checkout',
-                                  '--force', '--quiet', revision)
+        if env:
+            build_cmd = env.run(
+                'git', 'checkout',
+                '--force', '--quiet', revision,
+                cwd=self.working_dir,
+            )
+            code, out, err = build_cmd.exit_code, build_cmd.output, build_cmd.error
+        else:
+            code, out, err = self.run('git', 'checkout',
+                                      '--force', '--quiet', revision)
```

Comment:
Based on your screenshots, `--quiet` is not what we want it seems. The output without `--quiet` was really helpful. So 👍  on removing that from the command.